src/CoreBundle/private/library/classes/dbobjects/TCMSTableConf.class.php

```diff
@@ -152,11 +152,20 @@ public function GetCmsTblListExtension($sExtensionName)
      */
     public function &GetFields(&$oTableRow, $loadDefaults = false, $bDoNotUseAutoObjects = false)
     {
+        // TODO this could also be a parameter here?
+        $languageService = self::getLanguageService();
+        $language = $languageService->getActiveEditLanguage();
+
         $oFieldDefinition = &$this->GetFieldDefinitions(array(), $bDoNotUseAutoObjects);
         $oFields = new TIterator();
-        while ($oFieldDef = $oFieldDefinition->Next()) {
+        while ($oFieldDef = $oFieldDefinition->next()) {
```

Comment:
I saw only now that PHP-CS-Fixer lower-cased that method name, which was not intended (I filed an issue: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4218).

Not sure what to do. Method names are case-insensitive in PHP, and the lower-cased name conforms to our coding standards, but I don't like surprise features that much. Leave it for now, but maybe we'll revert to upper-case.

Reply:
next() and current() are called like that in TIterator (since first cs-fixer run there probably).

So this here seems to be ok (and no bug).

Reply:
That's what I meant: PHP-CS-Fixer lower-cased the methods in TIterator when we applied it to the code base last year. Your change can stay as is.